lib/cocoapods/installer.rb

```diff
@@ -594,6 +594,8 @@ def prepare_pods_project
           build_configuration.build_settings['TVOS_DEPLOYMENT_TARGET'] = tvos_deployment_target.to_s if tvos_deployment_target
           build_configuration.build_settings['STRIP_INSTALLED_PRODUCT'] = 'NO'
           build_configuration.build_settings['CLANG_ENABLE_OBJC_ARC'] = 'YES'
+          build_configuration.build_settings['CODE_SIGNING_REQUIRED'] = 'NO'
+          build_configuration.build_settings['PROVISIONING_PROFILE_SPECIFIER'] = 'AAAAAAAAAA/'
```

Comment:
Open up and say `AAAAAAA\`

![Giphy](http://media3.giphy.com/media/JavtlJ2FXJbnq/giphy.gif)